scripts/build.d/numpy

```diff
@@ -26,16 +26,16 @@ if [ $(uname) == Darwin ] ; then
   cat >> site.cfg << EOF
 [openblas]
 libraries = openblas
-library_dirs = ${DEVENVPREFIX}/lib
-include_dirs = ${DEVENVPREFIX}/include/openblas
+library_dirs = ${DEVENVPREFIX}/lib:/usr/local/opt/openblas/lib:/usr/local/lib:/usr/lib
```

Comment:
Add other locations for openblas: devenv, homebrew, local install, and system lib.

I am not sure about the order of homebrew and local install.  Any comments?

Reply:
In my opinion, I would say homebrew should have higher priority because `homebrew` is "more customized" solution than packages installed in `/usr/local`.

Reply:
Ok.  Let me keep this order if no one else objects.